README.md

```diff
@@ -44,7 +44,7 @@ The target is to provide a performant well-tested implementation of a wide range
 | multistream-select | /multistream/1.0.0 | ✅             |
 | plaintext          | /plaintext/2.0.0   | ✅             |
 | noise              | /noise             | ✅             |
-| tls                | /tls/1.0.0         | ⬜ help wanted |
```

Comment:
Could you add spaces to make it aligned with other rows?

Reply:
done!